samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -472,6 +495,7 @@ def _is_debugging(self) -> bool:
 
     def _get_stacks(self) -> List[Stack]:
         try:
+            print(SamLocalStackProvider.get_stacks)
```

Comment:
Left over debugging extra `print` statement here, lets remove it

Reply:
Done.